src/libraries/System.Private.CoreLib/src/System/Number.Parsing.cs

```diff
@@ -50,7 +50,7 @@ internal interface IBinaryIntegerParseAndFormatInfo<TSelf> : IBinaryInteger<TSel
     internal interface IBinaryFloatParseAndFormatInfo<TSelf> : IBinaryFloatingPointIeee754<TSelf>, IMinMaxValue<TSelf>
         where TSelf : unmanaged, IBinaryFloatParseAndFormatInfo<TSelf>
     {
-        static abstract int NumberBufferLength { get; }
+        static abstract int NumberBufferLength { get; } // Ceiling(Log10(5^(Abs(MinBinaryExponent) - 1))) + NormalMantissaBits + 1 + 1
```

Comment:
nit: These would be more useful as xml comments. It makes them a little more visible